docs/pages/push-notifications/using-fcm.md

```diff
@@ -45,6 +45,7 @@ Note that FCM is not currently available for Expo iOS apps.
   ```
 
   > **Note:** Firebase will create an API key in the Google Cloud Platform console with a name like `Android key (auto created by Firebase)`. **This is not always the same key as the one found in `google-services.json`. Always confirm your key and associated restrictions in the Google Cloud Platform console.**
+  > **Note:** The google-services.json file generated by Firebase may include a trailing comma after the current_key entry.  **This can cause a MalformedJsonException during the expo build:android proces.**.  Simply remove the trailing comma to avoid this exception.
```

Comment:
```suggestion
  > **Note:** The google-services.json file generated by Firebase may include a trailing comma after the current_key entry.  **This can cause a MalformedJsonException during the expo build:android process.**.  Simply remove the trailing comma to avoid this exception.
```

Reply:
I can't reproduce this. Could it be that this was an intermittent bug on Google's side? Could you try downloading your google-services.json file again to check if it is there?